draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,107 +32,208 @@ email = "tomh@apnic.net"
+    "country": "AU",
+    "parentHandle": "YYYY-RIR",
+    "status": [ "active" ],
+    "links":
+     [
+        {
+            "value": "https://example.net/ip/2001:db8::/48",
+            "rel": "self",
+            "href": "https://example.net/ip/2001:db8::/48",
+            "type": "application/rdap+json"
+        },
+        {
+            "value": "https://example.net/ip/2001:db8::/48",
+            "rel": "geofeed",
+            "href": "https://example.net/geofeed",
+            "type": "application/geofeed+csv"
+        },
+        ...
+    ],
+    ...
+}
+```
+
+# Redaction
+
+Since the "geofeed" link objects in the "links" array of an IP Network object are optional, the Redaction by Removal
+method [@?I-D.ietf-regext-rdap-redacted] MUST be used when redacting them. The following is an elided example of an IP
+Network object with redacted "geofeed" links:
+
+```
+{
+    "objectClassName": "ip network",
+    "handle": "XXXX-RIR",
+    "startAddress": "2001:db8::",
+    "endAddress": "2001:db8:0:ffff:ffff:ffff:ffff:ffff",
+    "ipVersion": "v6",
+    "name": "NET-RTR-1",
+    "type": "DIRECT ALLOCATION",
+    "country": "AU",
+    "parentHandle": "YYYY-RIR",
+    "status": [ "active" ],
+    "links":
+     [
+        {
+            "value": "https://example.net/ip/2001:db8::/48",
+            "rel": "self",
+            "href": "https://example.net/ip/2001:db8::/48",
+            "type": "application/rdap+json"
+        },
+        ...
+    ],
+    "redacted":
+    [
+        {
+            "name":
+            {
+                "description": "Geofeed links"
+            },
+            "prePath": "$.links[?(@.rel=='geofeed')]",
+            "method": "removal"
+        }
+    ],
+    ...
 }
 ```
 
 # Privacy Considerations
 
-When including a geofeed file URL in an IP Network object, an RDAP server
-operator SHOULD follow the guidance from [@?I-D.ymbk-opsawg-9092-update, section 7]
-to not accidentally expose the location of an individual. 
+When including a geofeed file URL in an IP Network object, an RDAP server operator SHOULD follow the guidance from
+[@?I-D.ymbk-opsawg-9092-update, section 7] to not accidentally expose the location of an individual. 
 
 # Security Considerations
 
-[@?I-D.ymbk-opsawg-9092-update] requires an HTTPS URL for a geofeed file, and
-optionally RPKI-signing the data within. Besides that, this document does not
-introduce any new security considerations past those already discussed in the
+[@?I-D.ymbk-opsawg-9092-update] requires an HTTPS URL for a geofeed file, and optionally RPKI-signing the data within.
+Besides that, this document does not introduce any new security considerations past those already discussed in the
 RDAP protocol specifications.
 
 # IANA Considerations
 
-IANA is requested to register the following value in the RDAP Extensions
-Registry:
+## RDAP Extensions Registry
+
+IANA is requested to register the following value in the RDAP Extensions Registry:
```

Comment:
Good observation. I think we should for now still have an extension identifier, albeit it'd be more like a "marker" extension per the definitions in the RDAP Extensions draft. Also, feel more comfortable having it in case we ever evolve it for other representations beside "application/geofeed+csv".